run.sh

```diff
@@ -20,7 +20,14 @@ usage() {
 case "$CMD" in
     "test" )
         echo "Django version: $(python -m django --version)"
-        python -m django test django_ratelimit "$@"
+        python \
+            -W error::ResourceWarning \
+            -W error::DeprecationWarning \
+            -W error::PendingDeprecationWarning \
```

Comment:
Is this only targeted for the Django 4.2 tests? Just thinking that it's a shame in Django 4.3 to miss new warnings.

Reply:
Django's `RemovedInDjangoXYWarning` warnings are floating between `DeprecationWarning` and `PendingDeprecationWarning` depending on the current version: https://github.com/django/django/blob/main/django/utils/deprecation.py#L7-L12

(Also there won't be a Django 4.3, just a 5.0)

Reply:
Ah! I read it wrongly, I'm too used to these patterns always being about silencing warnings. This is the thing we want :100: